src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -341,7 +341,7 @@ class WorkspaceMembersPage extends React.Component {
                             />
                         </View>
                         <View style={[styles.w100, styles.mt4, styles.flex1]}>
-                            <View style={[styles.peopleRow]}>
+                            <View style={[styles.peopleRow, styles.ph5, styles.pb5]}>
```

Comment:
Perhaps we can reduce this just a bit, maybe `.pb3` will be fine?

Reply:
styles.pb3 seems good to me kindly have a look at them.

Web :- 

https://user-images.githubusercontent.com/81307212/200699350-17b578af-ff0a-4f4b-9034-5d15f7f42336.mov

Desktop :- 

https://user-images.githubusercontent.com/81307212/200699376-890bf07a-3158-425c-af32-0d27c994aba7.mov

IOS :-


https://user-images.githubusercontent.com/81307212/200701018-c9e33880-0280-439c-b059-2a56b76d51e2.mov


Android :- 

https://user-images.githubusercontent.com/81307212/200700879-1d72a85e-33ad-4f6f-a09c-13899b929cf9.mp4

If it is okay I will push the code.

Reply:
Looks great, let's do it! Thanks!